tests/conftest.py

```diff
@@ -132,8 +132,8 @@ def disable_pip_version_check():
 
 
 @pytest.fixture(scope='session')
-def pep518_wheelhouse(tmpdir_factory):
-    wheelhouse = tmpdir_factory.mktemp('wheelhouse')
+def pep518_wheelhouse(tmp_path_factory):
+    wheelhouse = tmp_path_factory.mktemp('wheelhouse')
```

Comment:
Can you please check if we make use of `tmpdir_factory` anywhere else? I have the feeling that I cargo-culted this fixture elsewhere...

Reply:
Done, got rid of all usage of (pytest) tmpdir.

Reply:
Thank you!